src/NHibernate/Multi/QueryBatchItemBase.cs

```diff
@@ -16,6 +16,8 @@ namespace NHibernate.Multi
 	/// </summary>
 	public abstract partial class QueryBatchItemBase<TResult> : IQueryBatchItem<TResult>, IQueryBatchItemWithAsyncProcessResults
 	{
+		private static readonly INHibernateLogger Log = NHibernateLogger.For(typeof(QueryBatch));
```

Comment:
`QueryBatch` type here is intentional.. Don't want to create separate logger for each result type